### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by AGindinson**
This brings in a chaotic bunch of doc improvements:
- grammar fixes;
- minor stylistis fixes;
- trailing whitespaces' removal.

Signed-off-by: Artem Gindinson <amgindinson@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,33 +2,36 @@
...
-- prepare users info  and place it in `meta_files/users_data`
-- add users  to DB with `  ./manage.py add_users`
-this command would print credentials for all users from student and staff  groups.  
-It will also create one user with extra privileges called bank_manager. 
-- collect static files from different apps to single folder `./manage.py collectstatic`
-- run test server  `  ./manage.py runserver`
--  go to  <http://127.0.0.1:8000/bank>  and enter credentials printed in console to test app
+```
+- apply migrations: `  ./manage.py migrate`
+- create superuser for admin actions: `  ./manage.py createsuperuser`
```
Comment (AGindinson) [inline README.md]:
```suggestion
- create a superuser for admin actions: `  ./manage.py createsuperuser`
```

File: README.md [hunk 2]
```diff
@@ -2,33 +2,36 @@
...
+- populate static tables: `  ./manage.py add_static_data`
+- prepare users' info and place it in `meta_files/users_data`
+- add users to the DB with `  ./manage.py add_users`
+This command will print credentials for all users from student and staff groups.  
+It will also create one user with extra privileges called *bank_manager*.
```
Comment (AGindinson) [inline README.md]:
```suggestion
It will also create one user with extra privileges called `bank_manager`.
```

File: README.md [hunk 3]
```diff
@@ -39,41 +42,41 @@ python ./manage.py test
 
 ### Building with docker
 
-build image for local hosting with
+Build image for local hosting with:
```
Comment (AGindinson) [inline README.md]:
```suggestion
Build an image for local hosting with:
```

**Comment by nkorobkov** [comment]
Also, if while deploying you found that some steps are not clear enough, feel free to change the wording or add additional paragraphs as you see fit.

**Comment by nkorobkov** [comment]
Also, if while deploying you found that some steps are not clear enough, feel free to change the wording or add additional paragraphs as you see fit.

**Pull request merged**